inc/question.class.php

```diff
@@ -533,6 +533,44 @@ protected function deserializeDefaultValue($input) {
       return $input;
    }
 
+   public function moveUp() {
+      $order         = $this->fields['order'];
+      $sectionId     = $this->fields['plugin_formcreator_sections_id'];
+      $otherItem = new static();
+      $otherItem->getFromDBByQuery("WHERE `plugin_formcreator_sections_id` = '$sectionId'
```

Comment:
I have to update the found item, then it appears to me better to retrieve directly the object to update it a few lines later.